packages/cra-template-seed/template/post_create.js

```diff
@@ -0,0 +1,75 @@
+const fs = require('fs');
+
+const PACKAGE_FILE = 'package.json';
+const UTF_8 = 'utf-8';
+
+const husky = {
```

Comment:
why can't you add husky to the template.json?

Reply:
basically we have added the postinstall script into the package.json